packages/interactivity/src/test/store.ts

```diff
@@ -202,11 +213,18 @@ describe( 'Interactivity API', () => {
 							sync( n ) {
 								return n;
 							},
-							*async( n ): Generator< unknown, number, number > {
+							*async( n ): AsyncAction< number > {
 								const n1: number =
```

Comment:
And here 😊 

```suggestion
								const n1 =
```

Reply:
I've changed it to `satisfies` in https://github.com/woocommerce/gutenberg/pull/15/commits/b024e68c4e54a5ec4262299a607d23d03abc951f